### ISSUE TITLE
The package could not be found libgl1-mesa-dev-hwe-16.04

### ISSUE THREAD

**Issue by HernanA4Q** [openframeworks/openFrameworks#5974]
Hi,
I am installing OpenFrameworks for the first time in Ubuntu 16.04 LTS 64bits.
Before installing the dependencies, perform update and upgrade.
`sudo apt-get update`
`sudo apt-get upgrade`

Then order to install the dependencies.
`sudo ./install_dependencies.sh`
It tells me that the package libgl1-mesa-dev-hwe-16.04 was not found.

Part of the output per terminal is as follows:

> installing OF dependencies with -hwe-16.04 packages, confirm Y/N ? Y
installation of OF dependencies with -hwe-16.04 packages confirmed
installing OF dependencies
OF needs to install the following packages using apt-get:
curl libjack-jackd2-0 libjack-jackd2-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++ libgl1-mesa-dev-hwe-16.04 libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk-3-dev libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-libav gstreamer1.0-pulseaudio gstreamer1.0-x gstreamer1.0-plugins-bad gstreamer1.0-alsa gstreamer1.0-plugins-base gstreamer1.0-plugins-good premake4
Do you want to continue? [Y/n] Y
>
>Installing...
>
>Installing curl
 Already installed
Installing libjack-jackd2-0
 Already installed
Installing libjack-jackd2-dev
Installing freeglut3-dev
Extrayendo plantillas para los paquetes: 100%
Installing libasound2-dev
Installing libxmu-dev
Installing libxxf86vm-dev
 Already installed
Installing g++
 Already installed
Installing libgl1-mesa-dev-hwe-16.04
E: No se ha podido localizar el paquete libgl1-mesa-dev-hwe-16.04
E: No se pudo encontrar ningún paquete usando «*» con «libgl1-mesa-dev-hwe-16.04»
E: No se pudo encontrar ningún paquete con la expresión regular «libgl1-mesa-dev-hwe-16.04»
error installing libgl1-mesa-dev-hwe-16.04
Leyendo lista de paquetes... Creando árbol de dependencias... Leyendo la información de estado...
this seems an error with your distribution repositories but you can also
report an issue in the openFrameworks github: http://github.com/openframeworks/openFrameworks/issues

What should I do?
Thank you very much.

**Comment by gorkacortazar** [comment]
If i recall correctly it happens in 0.9.x, I tried saying N works without problem

**Comment by HernanA4Q** [comment]
Thank you very much!
Works 0.9.8!

Yes, there is no need to install libgl1-mesa-dev-hwe-16.04 in Ubuntu 16.04.

Sorry, after publishing, I found that this problem had already been mentioned [here](https://github.com/openframeworks/openFrameworks/issues/5873).

Later, when I reinstalled the dependencies, I had an error with "Poco".
But I could solve it thanks to the [forum](https://forum.openframeworks.cc/t/install-dependencies-sh-problem-poco-failed-to-recompile-ubuntu-16-04-x86/25182).

I had to install curl
`sudo apt-get install curl`

Well, thank you again!
Regards!
